docs/howtoguides/use_https_in_https_proxy.rst

```diff
@@ -0,0 +1,65 @@
+How to use an HTTPS proxy that uses HTTPS itself (HTTPS-in-HTTPS)
+*****************************************************************
+
+.. note:: Support for HTTPS-in-HTTPS proxying was added in version 28.1
+
+.. warning:: HTTPS-in-HTTPS is only supported by the Ubuntu Pro Client on Ubuntu 18.04 and later
+
+If you need to set ``https_proxy`` to a proxy that uses ``https://``, then you need to follow a couple extra steps to ensure all Ubuntu Pro actions work correctly.
+
+
+Install extra pycurl dependency
+===============================
+
+``pro`` switches to use ``pycurl`` only for HTTPS-in-HTTPS proxying, but ``pycurl`` may not be installed by default on your machine. To install it, run:
+
+.. code:: bash
+
+   sudo apt install python3-pycurl
+
+.. note::
+   If you don't do this and try to set the HTTPS-in-HTTPS proxy anyway, you will get an error that looks like this:
+
+   .. code:: none
+
+      To use an HTTPS proxy for HTTPS connections, please install pycurl with `apt install python3-pycurl`
+
+Ensure the proxy's HTTPS certificate will be trusted by Livepatch
+=================================================================
+
+.. tip:: You can skip this step if you don't plan on using Livepatch
```

Comment:
```suggestion
.. tip:: You can skip this step if you are not using Livepatch.
```

Reply:
Setting up the proxy pretty much always happens before attaching and enabling services, so the likelihood that someone is already using livepatch when going through this guide is very small -- so I'm not sure about this change.

Reply:
That makes sense then, feel free to ignore my comment